### ISSUE TITLE
New virology cure table reverts to first rolled cure of a unique symptom combination upon creating new culture bottles.

### ISSUE THREAD

**Issue by glubtok** [tgstation/tgstation#46014]
**Problem Description:** Culture bottles created after rerolling a unique symptom combination for a different cure will revert back to the first rolled cure for that unique symptom combination.

**What did you expect to happen:** The cure for the culture bottles created in the pandemic should have their cure as what the pandemic shows.

**What happened instead:** The cure for the cultures in the newly created bottles was that of the first cure ever rolled for that unique symptom combination.

**Why is this bad/What are the consequences:** Normally you infect yourself with the cure already in hand to make yourself immune to the disease as an antagonist before spreading it. The issue is, you'll end up unspectacularly cutting your antag round short because the cure is now different than what the pandemic showed. It also limits your ability to reroll to a different cure in a given tier by recreating the virus. This is better explained in the next section.

**Steps to reproduce the problem:** Create a unique virus from combining symptoms. Let's say we're pulling from cure-tier 4. Spaceacillin, epinephrine, charcoal and saline-glucose. The first time this unique set of symptoms is combined in a bottle, we get charcoal as the cure, for example. Of this tier level, spaceacillin is definitely the hardest to make, so being evil virologists, we'll keep trying to remake the disease until it rolls it as the cure. Once we get a culture remade in a beaker with our desired cure of spaceacillin, we'll go to reproduce it by creating another culture bottle. This is where the bug lies. The first ever cure rolled by this symptom combination was charcoal. Any subsequent bottles created for this unique symptom combination will always revert to charcoal, regardless of the cure rolled for the culture currently in the pandemic. 

The culture in the pandemic will correctly function as it says, however. If it says spaceacillin, you will require spaceacillin as the cure if you take the beaker out and infect yourself. You just can't create new bottles from the pandemic with anything but the first-rolled cure.

tl;dr: 
-Create unique symptom combination.
-Recreate identical virus through reagents or combining cultures to roll a different cure.
-Create bottles in pandemic from culture with the new cure. 
-These created bottles will all default back to the first ever rolled cure for that unique symptom combination.

**Server:** Both the most recent TG master branch hosted locally and another TG-based server (fulpstation) exhibit this bug.

**Revision:** All revisions after the July 3rd virology cure table update.

**Comment by kriskog** [comment]
Huh. So the way diseases work any unique combination has an ID , and any identical symptom combination will be the same disease for all intents and purposes - including the cure and vaccine and so on. This is however only checked when making a culture bottle, which is what causes this behaviour.
